### PR TITLE
chore: call close framework when finishing

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
- call close method when finishing framework
- `fake.NewSimpleClientset` method has been deprecated in favor of `NewClientset`.
#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Comment by googs1025** [comment]
@sanposhiho friendly ping /PTAL

File: pkg/scheduler/framework/runtime/framework_test.go [hunk 1]
```diff
@@ -913,8 +913,10 @@ func TestNewFrameworkMultiPointExpansion(t *testing.T) {
 					t.Fatalf("Unexpected error, got %v, expect: %s", err, tc.wantErr)
 				}
 			}
-
 			if tc.wantErr == "" {
+				defer func() {
```
Comment (sanposhiho) [inline pkg/scheduler/framework/runtime/framework_test.go:917]: better to place defer before L916?

```go
defer func()...
if tc.wantErr == "" {
```

**Review by sanposhiho — COMMENTED** [review]
only one nit

**Review by sanposhiho — APPROVED** [review]
/lgtm
/approve

**Pull request merged**